modules/core/src/test/kotlin/datamaintain/core/DatamaintainConfigTest.kt

```diff
@@ -30,19 +32,22 @@ class DatamaintainConfigTest {
                 FakeDriverConfig())
         expectThat(config) and {
             get { identifierRegex.pattern }.isEqualTo(CoreConfigKey.SCAN_IDENTIFIER_REGEX.default)
+            get { executionMode }.isEqualTo(ExecutionMode.NORMAL)
         }
     }
 
     @Test
     fun `should be overridden by jvm`() {
         System.setProperty("scan.path", "/new")
+        System.setProperty(CoreConfigKey.EXECUTION_MODE.key, "NORMAL")
 
         val config = DatamaintainConfig.buildConfig(DatamaintainConfigTest::class.java.getResourceAsStream("/config/default.properties"),
                 FakeDriverConfig())
         expectThat(config).and {
             get { path }.isEqualTo(Paths.get("/new"))
             get { identifierRegex.pattern }.isEqualTo("(.*?)_.*")
             get { blacklistedTags }.isEqualTo(setOf(Tag("un"), Tag("deux")))
+            get { executionMode }.isEqualTo(ExecutionMode.NORMAL)
```

Comment:
Since the default value of execution mode is "NORMAL", trying to override it with another value and check that it worked seems more pertinent to me.

Reply:
No for the same reason  DTMT-17. We can speak about that together.

Reply:
Oh alright, the value given by the properties file is "DRY" and you're checking that it was overridden! I get it now!